.github/workflows/sql-workbench-test-and-build-workflow.yml

```diff
@@ -25,9 +25,10 @@ jobs:
           path: sql/OpenSearch-Dashboards
 
       - name: Setup Node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
-          node-version: '14.18.2'
+          node-version-file: ".nvmrc"
```

Comment:
I think you have to point to the OpenSearch Dashboards directory to get this to work.

Reply:
You beat me to it :)